src/vcsparser.core/DailyCodeChurnBugDatabase.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace vcsparser.core
+{
+    public class DailyCodeChurnBugDatabase
+    {
+        public int NumberOfChanges { get; set; }

```

Comment:
This is unnecessary. Everything that is in the bug database is NumberOfChangesInFixes.

Reply:
Renamed to `NumberOfChangedInFixes` in 436638d125bb40b11223df403d8dd9b234d55800
This was not removed as there is a measure that is using it (`NumberOfChangesInFixesBugDatabaseMeasureAggregator.cs`)